index.bs

```diff
@@ -408,14 +408,16 @@ Excluding inputs generally include
 However, an event whose only effect is to begin or update a fling or scroll
 gesture is not an excluding input.
 
-The user agent may delay the reporting of layout shifts after a
-<a href="https://www.w3.org/TR/pointerevents/#the-pointerdown-event">pointerdown</a> event
-until such time as it is known that the event does not begin a fling or scroll
-gesture.
-
-The <a href="https://www.w3.org/TR/uievents/#event-type-mousemove">mousemove</a> and
-<a href="https://www.w3.org/TR/pointerevents/#the-pointermove-event">pointermove</a>
-events are also not excluding inputs.
+* When a <a href="https://www.w3.org/TR/pointerevents/#the-pointerdown-event">pointerdown</a> event or
+    a <a href="https://www.w3.org/TR/uievents/#event-type-mousedown">mousedown</a> event is received, the
+    user agent starts buffering the layout shifts.
+* When a <a href="https://www.w3.org/TR/pointerevents/#the-pointercancel-event">pointercancel</a> event
+    is received, all the accumulated layout shift score in the buffer is reported, and it is not an excluding input.
+* When a <a href="https://www.w3.org/TR/pointerevents/#the-pointerup-event">pointerup</a> event or a
+    <a href="https://www.w3.org/TR/uievents/#event-type-mouseup">mouseup</a> event is received, the buffer is reset.
```

Comment:
nit: 'buffer is reset' is a bit unclear since we didn't say what the buffer is. But in any case we need to report the accumulated layout shifts before that. Do we need to also reset when we get pointercancel?

Also, we may want to specify how we compute attribution while buffering? I actually think I found a Chrome bug, filing it.